### PR TITLE
API 1711 observaton token search

### PR DESCRIPTION
**Pull request by bschofield-va**

### CODE REVIEW

File: data-query/src/test/java/gov/va/api/health/dataquery/service/controller/procedure/Dstu2ProcedureControllerTest.java [hunk 1]
```diff
@@ -34,14 +34,13 @@
...
-import org.springframework.test.annotation.DirtiesContext;
 import org.springframework.test.context.junit.jupiter.SpringExtension;
 
 @DataJpaTest
-@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
 @ExtendWith(SpringExtension.class)
 public class Dstu2ProcedureControllerTest {
-  HttpServletResponse response = mock(HttpServletResponse.class);;
+  HttpServletResponse response = mock(HttpServletResponse.class);
+  ;
```
Comment (jhulbert-va) [inline data-query/src/test/java/gov/va/api/health/dataquery/service/controller/procedure/Dstu2ProcedureControllerTest.java:43]: ?

File: data-query/src/test/java/gov/va/api/health/dataquery/service/controller/Dstu2BundlerTest.java [hunk 2]
```diff
@@ -30,7 +30,8 @@
       x -> FugaziArgo.of(x.id());
 
   PageLinks links = mock(PageLinks.class);
-  Dstu2Bundler bundler = new Dstu2Bundler(links);;
+  Dstu2Bundler bundler = new Dstu2Bundler(links);
+  ;
```
Comment (jhulbert-va) [inline data-query/src/test/java/gov/va/api/health/dataquery/service/controller/Dstu2BundlerTest.java:34]: ?

File: data-query/src/test/java/gov/va/api/health/dataquery/service/controller/observation/R4ObservationControllerTest.java [hunk 3]
```diff
@@ -799,9 +800,46 @@ void searchByPatientAndCategorySystemAndCode() {
...
+    Multimap<String, Observation> observationsByPatient = populateData();
+    assertThat(
+            json(
+                controller()
+                    .searchByPatientAndCategory(false, "p0", "http://nope.com|", null, 1, 10)))
+        .isEqualTo(
+            json(
+                ObservationSamples.R4.asBundle(
+                    "http://fonzy.com/cool",
+                    Collections.emptyList(),
```
Comment (jhulbert-va) [inline data-query/src/test/java/gov/va/api/health/dataquery/service/controller/observation/R4ObservationControllerTest.java:814]: Statically import this?

File: data-query/src/test/java/gov/va/api/health/dataquery/service/controller/procedure/Dstu2ProcedureControllerTest.java (hunk 1 above)
Comment (bschofield-va) [reply data-query/src/test/java/gov/va/api/health/dataquery/service/controller/procedure/Dstu2ProcedureControllerTest.java:43]: google formatter changes? either way should not have add `;;`

Comment (jblefkove-va) [reply data-query/src/test/java/gov/va/api/health/dataquery/service/controller/procedure/Dstu2ProcedureControllerTest.java]: Surprised checkstyle doesn't complain about double semicolons

**Pull request merged**